fastmri_examples/feature_varnet/pl_modules/__init__.py

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
This folder should have a different name and potentially gives a namespace conflict. Could you move `feature_varnet_module.py` to the top level and remove this folder? Then in `train_feature_varnet.py` import the module from `feature_varnet_module.py`.